UnityProject/Assets/Scripts/Learning/ProtipObjectTypes/ProtipObjectOnEnterRadius.cs

```diff
@@ -25,15 +28,16 @@ private void CheckForNearbyItems()
 			foreach (var target in possibleTargets)
 			{
 				if(gameObject == target.gameObject) continue;
-				if (MatrixManager.Linecast(gameObject.RegisterTile().WorldPosition, LayerTypeSelection.Walls,
+				
+				if (MatrixManager.Linecast(Vector3.ClampMagnitude(gameObject.RegisterTile().LocalPositionClient, SearchRadius), LayerTypeSelection.Walls,
```

Comment:
I think this should clamp the difference between the object and the target.

Reply:
Linecast takes two world positions, not a local position.

Reply:
Fixed it and I found a solution to stopping the linecast from causing errors by using `Vector3.Distance()` before calling that funciton